### ISSUE TITLE
ScavengerBot token

### ISSUE THREAD

**Issue by itguypw** [ScavengerBot/ScavengerBot#6]
A scavenger bot utility token would be awesome to pair with the bot in the future as an option to pay fees and have the community "invest" in the bot.

**Comment by Crypto2legit** [comment]
We have been asked this a couple times. Thank you for showing the interest. Do you have any other thoughts on this? We like the idea!

**Comment by Crypto2legit** [comment]
We have added this to our discussions thank you

**Comment by itguypw** [comment]
It's great that you are adding this to the discussion. I think a utility
token would be a cool aspect since the bot is crypto centric. Having the
user community invest into a token that represents the bot could have a few
positive impacts such as: a feeling of involvement from the supporting
community by owning the tokens, the token as a secondary value generator
for the developers/owners, marketing for the bot as the token gets
exchanged, and who knows what else.
I think there are a few different things you can do with the tokens. The
first and obvious is to allow it as a source of payment for the monthly
fees, similar to services like storj. You would be able to do airdrops, or
contests, or dividends, any number of cool things.
I would not mind being part of the discussion if you don't mind me hearing
what you think or providing ideas if asked.

On Nov 5, 2018 4:43 PM, "2Legit" <notifications@github.com> wrote:

We have been asked this a couple times. Thank you for showing the interest.
Do you have any other thoughts on this? We like the idea!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/ScavengerBot/ScavengerBot/issues/6#issuecomment-436063492>,
or mute the thread

**Comment by Crypto2legit** [comment]
This is good information to know. This is been asked now a couple times i wonder if this is something we can gauge better. Since we are a community maybe a poll? We are of course open to options! Are you suggesting token holders would vote on development, ect?

**Comment by itguypw** [comment]
Giving token holders a say would be up to you guys. To treat them like
share holders, is one possible side benefit to the token...
Personally, I think if the core team is doing a good job then you don't
want to many outside voices giving their opinions or having too many cooks
in the kitchen so to speak. I would not give token holders a say in the
development.
Here's a trick, if you really think voting power would give the token some
appeal then you don't open everything to a vote. You only open things to a
vote that the decision is impactful. Here's an example: the Beta team has
been testing the ETH and BCH pairs and think they are both close, vote to
see which gets released first.  Or which of 2 exchanges to add that are
already on your list of target exchanges..

Back to my thoughts on the token:
Say you release tokens, users must pay the fee with the tokens, so user
must buy tokens and creates a demand.
The user pays the fees and now the dev team has SAV tokens instead of BTC
or USD. The users would pay the same amount of fees it would just be the
exchange equiv of the BTC fee in SAV, say 5000 SAV =1 BTC just for the
example.. my bot did well and my fee would be .01 BTC I would pay the Devs
50 SAV. The devs could then exchange those for the .01 BTC or hold them or
what ever they choose.
But as the bot gets more popular, more people will need to be buying the
SAV to have to pay the fees each month. Some people may want to buy the
token just to show support in the project, some people just like having
portfolios of utility tokens... So as people begin buying the tokens it
creates more demand and should increase the value of the tokens.
Month 2 my SAV may be worth 4000= 1 BTC so I would only need to send 40SAV
to pay the .01 BTC fee, but the 50 SAV I sent the first month is now worth
.012 BTC if the devs held the tokens.
This cycle would just continue as the bot became more popular, and the
token could even be traded or invested in by people that were not even
users of the bot themselves.

The bottom line I think the token and the bot would compliment each other
in both cross marketing each other and increasing value for the users.

**Comment by itguypw** [comment]
I also want to point out a added benefit to the core Scavenger Bot team for
creating and releasing a token. I'm going to say that I'm not sure how the
team is getting paid beside the potential monlthy fees.
But if you release a token you have the option to pre distribute a portion
of the tokens to the core team. Just say 10,000 to each as an example. Now
the core team has the potential of the monthly fees and unlimited potential
for the future value of their 10,000 tokens.
